libsolutil/IpfsHash.cpp

```diff
@@ -55,34 +55,125 @@ string base58Encode(bytes const& _data)
 }
 }
 
+struct Chunk
```

Comment:
Why not put it inside the anonymous namespace?

Reply:
Fixed